### ISSUE TITLE
--var integer value passed is turned to floating point value

### ISSUE THREAD

**Issue by winni0713** [vmi/selenese-runner-java#258]
When passing --var (-V) \<var-name\>=\<json-value\>  , integer value (e.g. "80") is turned to floating point value (i.e. "80.0")

command: '..... --var myValue="80" '
```
<tr>
	<td>echo</td>
	<td>${myValue}</td>
        <td></td>
</tr>
```
output: "80.0" is printed out

**Comment by vmi** [comment]
Hi,

I fixed this problem, and released new version 3.9.0.
Please test it.

**Comment by vmi** [comment]
Please re-open this issue if your problem is not resolved.
